app/assets/javascripts/3dbio_viewer/src/webapp/components/RootViewerContents.tsx

```diff
@@ -41,6 +44,14 @@ export const RootViewerContents: React.FC<RootViewerContentsProps> = React.memo(
     const networkToken = selection.type === "network" ? selection.token : undefined;
     const proteinNetwork = externalData.type === "network" ? externalData.data : undefined;
 
+    const toggleToolbarExpanded = React.useCallback(
+        (_e: React.SyntheticEvent, data: ResizeCallbackData) => {
+            setToolbarExpanded(data.size.width >= 520);
+            setViewerSelectorExpanded(window.innerWidth - data.size.width >= 725);
```

Comment:
There are some references to window.xyz in this component. But then, should we listen to viewport size changes. If I change the width, I don't see correct % applied (I think)

Reply:
Yea it doesn't change when the viewport changes the width. But if I add `window.innerWidth` as a dependecy it throws a warning:

> React Hook React.useCallback has an unnecessary dependency: 'window.innerWidth'. Either exclude it or remove the dependency array. Outer scope values like 'window.innerWidth' aren't valid dependencies because mutating them doesn't re-render the component

Is there any way to listen to viewport changes